assets/prototype/domain/eventEditor/ticketsList/ticketPriceCalculator/TicketPriceModifierRow.js

```diff
@@ -0,0 +1,156 @@
+import clone from 'ramda/src/clone';
+import drop from 'ramda/src/drop';
+import find from 'ramda/src/find';
+import propEq from 'ramda/src/propEq';
+import { Field } from 'react-final-form';
+import { Button } from '@blueprintjs/core';
+
+// just temporary
+import styles from './inlineStyles';
+
+// also temporary, needs to come from the db
+const allOptions = [
+	{ id: 1, type: 'Base Price', isDiscount: false, isPercent: false, order: 0 },
+	{ id: 2, type: 'Percent Discount', isDiscount: true, isPercent: true, order: 20 },
+	{ id: 3, type: 'Dollar Discount', isDiscount: true, isPercent: false, order: 30 },
+	{ id: 4, type: 'Percent Surcharge', isDiscount: false, isPercent: true, order: 40 },
+	{ id: 5, type: 'Dollar Surcharge', isDiscount: false, isPercent: false, order: 50 },
+	{ id: 6, type: 'Regional Tax', isDiscount: false, isPercent: true, order: 60 },
+	{ id: 7, type: 'Federal Tax', isDiscount: false, isPercent: true, order: 70 },
+];
+const modifierOptions = drop(1, allOptions); // removes first option
+const getBaseType = (type) => find(propEq('id', Number(type)))(modifierOptions);
+
+// need to change these based on site i18n config
+const currencySign = '$';
+const currencySignB4 = true;
+const decimalPlaces = 2;
+// const decimalMark = '.';
+// const thousandsSep = ',';
+const percentSign = '%';
+
+function formatPriceAmount(amount) {
+	return Number.parseFloat(amount).toFixed(decimalPlaces);
+}
+
+const TicketPriceModifierRow = ({ index, name, price, calcDir, fields: { push, remove, reset, sort } }) => {
+	const options = price.priceType === 1 ? allOptions : modifierOptions;
+	const sign = price.isPercent ? percentSign : currencySign;
+	let b4Price = '';
+	let afterPrice = sign;
+	// isPercent T  currencySignB4 T  sign appears  afterPrice
+	// isPercent T  currencySignB4 F  sign appears  afterPrice
+	// isPercent F  currencySignB4 F  sign appears  afterPrice
+	// isPercent F  currencySignB4 T  sign appears  b4Price
+	if (currencySignB4 && !price.isPercent) {
+		b4Price = sign;
+		afterPrice = '';
+	}
+	const actions = [];
```

Comment:
What about this:
```suggestion
	const Actions = () => (
	<>
		{price.id === 'NEW_PRICE' && (
			<Button
				key={'add'}
				icon={'add'}
				onClick={() => {
					const newPrice = clone(price);
					newPrice.id = null;
					const baseType = getBaseType(newPrice.priceType);
					newPrice.order = baseType.order;
					newPrice.isDiscount = baseType.isDiscount;
					newPrice.isPercent = baseType.isPercent;
					push(newPrice);
					reset(name);
					sort();
				}}
				minimal
			/>
		)}
		{!price.isBasePrice && (
			<Button
				key={'trash'}
				icon={'trash'}
				onClick={() => {
					remove(index);
				}}
				minimal
			/>
		)}
	</>
);


```

?